src/murmur/Server.cpp

```diff
@@ -527,6 +529,15 @@ void Server::readParams() {
 	if (iMessageBurst < 1) { // Prevent disabling messages entirely
 		iMessageBurst = 1;
 	}
+
+	iPluginMessageLimit = getConf("mpluginessagelimit", iPluginMessageLimit).toUInt();
+	if (iPluginMessageLimit < 1) { // Prevent disabling messages entirely
+		iPluginMessageLimit = 1;
```

Comment:
This seems counter intuitive: usuallly -1 or 0 means disable the functionality this setting is meant to control so it should not check at all (of course this is dangerous but admins usually have reasons for this). 

I suggest:
Either log an error message for <1 or disable the checking and dokument it as magic value to turn the protection off.

Reply:
This is actually mirrored from the original rate-limiter that is in place for all control messages passing through the server. I agree that there is a better way of handling these but imo this should be a follow-up PR that'll deal with both cases at once so that the rate-limiting stays in sync with how they can be used. 

And a value  <1 wouldn't actually turn the protection off. Instead it would block all messages without exception. That's why setting it to that value is not allowed.

Reply:
Ok. Now I see what you mean. The concept of the LeakyBucket metaphor didn't really get transferred by the variable names to me. 
I am now also seeing that this was already existing code. 

Of course one could not simply disable the limiting without switching away from LeakyBucket. 

That being said I think that my confusion originates from the transition of 'limit' (implying a total with the unit being messages) to 'leak rate' (rate: unit is mesages per second). While burst could imply being a rate when it is actually is a limit (not directly limeted to a the timeframe - only implicitly). 
I agree that this could be something for a different PR, if at all. 

But maybe I am overcomplicating it. 
Setting the limit to 1 would not limit the  messages

Reply:
I agree that the naming is quite confusing but given that its been like this since the introduction of the rate-limiter (afaik) I think most Mumble server admins know that by now and therefore I tend towards using the same naming for the plugin rate-limiting... :thinking: